admin/page.php

```diff
@@ -10,7 +10,7 @@
 
 		BackUpWordPress
 
-		<a class="page-title-action" href="<?php echo esc_url( https://bwp.hmn.md/ ); ?>" target="_blank">Extensions</a>
+        <a class="page-title-action" href="<?php echo esc_url( 'https://bwp.hmn.md/' ); ?>" target="_blank">Extensions</a>
```

Comment:
Escaping has always been something I get confused easily over, but this makes sense to me. Thanks!